### PR TITLE
Fix #133

### LINKED ISSUE #133: WebSocket Peers are not being deleted
Version 7.5.4

For the moment, that's all the information I have.

### PR DESCRIPTION
**Pull request by DiegoRBaquero**

### CODE REVIEW

**Comment by yciabaud** [comment]
The test passes, the previous travis build timed out for no reason.

File: server.js [hunk 1]
```diff
@@ -337,6 +337,15 @@ Server.prototype._onWebSocketRequest = function (socket, opts, params) {
     }
     if (self.destroyed) return
 
+    var hashes
+    if (typeof params.info_hash === 'string') hashes = [ params.info_hash ]
```
Comment (feross) [inline server.js]: This already happens in parse-websocket.js

File: server.js [hunk 2]
```diff
@@ -337,6 +337,15 @@ Server.prototype._onWebSocketRequest = function (socket, opts, params) {
...
+    else hashes = params.info_hash
+    hashes.forEach(function (info_hash) {
+      if (socket.infoHashes.indexOf(info_hash) === -1) {
+        socket.infoHashes.push(info_hash)
```
Comment (feross) [inline server.js]: This should only happen for announce. I will move it into the if statement belo

**Comment by feross** [comment]
7.5.5

**Pull request merged**